charts/dftbackend/templates/hpa.yaml

```diff
@@ -1,4 +1,23 @@
 {{- if .Values.autoscaling.enabled }}
+#################################################################################
+# Copyright (c) 2021, 2022, 2023 T-Systems International GmbH
+# Copyright (c) 2021,2022,2023 Contributors to the Eclipse Foundation
```

Comment:
license header could/should be the first thing in the file. I don't think it needs to be after the if

Reply:
While deploying the application , it gives us error in argocd groupVersion shouldn't be empty.
That why we have made changes like that.

Reply:
license header corrected.